src/pages/blog/blogArchives/blogArchives.scss

```diff
@@ -1,8 +1,16 @@
 #blog-archive-container {
-  display: flex;
-  flex-wrap: wrap;
   padding: 5em 3em;
   margin: 0 auto;
+
+  #blog-archive-header {
+    font-size: 3em;
```

Comment:
At 3em the title is too big on mobile screens. We want whole title on a single line. A size that seems to work for this on mobile is 2.5 em. You will probably need to add some media queries so that everything is sized correctly on tablet and desktop after reducing the size for mobile.